packages/scripts/README.md

```diff
@@ -114,14 +114,18 @@ _Example:_
 ```json
 {
 	"scripts": {
-		"lint:js": "wp-scripts lint-js ."
+		"lint:js": "wp-scripts lint-js",
+		"lint:js:src": "wp-scripts lint-js ./src"
```

Comment:
I like this, it ties in with what you mentioned here: https://github.com/WordPress/gutenberg/pull/15890#discussion_r288966458